languages/id.yml

```diff
@@ -0,0 +1,86 @@
+# format tanggal yang ditampilkan didalam blog
+date_format: "MMM DD, YYYY"
+
+global:
+    home: "Depan"
+    categories: "Kategori"
+    category: "Kategori"
+    tags: "Tag"
+    tag: "Tag"
+    archives: "Arsip"
+    search: "Telusuri"
+    about: "Tentang"
+    author_picture: "Foto penulis"
+    github: "GitHub"
+    stack_overflow: "Stack Overflow"
+    twitter: "Twitter"
+    facebook: "Facebook"
+    google_plus: "Google +"
+    weibo: "Weibo"
+    qq: "QQ"
+    qzone: "Qzone"
+    renren: "Renren"
+    vk: "VK"
+    odnoklassniki: "Odnoklassniki"
+    linkedin: "LinkedIn"
+    mail: "Mail"
+    rss: "RSS"
+    share_on_facebook: "Bagikan ke Facebook"
+    share_on_twitter: "Bagikan ke Twitter"
+    share_on_google_plus: "Bagikan ke Google+"
+    share_on_weibo: "Bagikan ke Weibo"
+    share_on_qq: "Bagikan ke QQ"
+    share_on_qzone: "Bagikan ke Qzone"
+    share_on_renren: "Bagikan ke Renren"
+    share_on_vk: "Bagikan ke VK"
+    share_on_odnoklassniki: "Bagikan ke Odnoklassniki"
+    search_category: "Telusuri kategori"
```

Comment:
`Telusuri`, would `mencari` be more appropriate? I see that `Telusuri` means `browse`.

Reply:
I think you are right